blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
```

Comment:
That later comment also means I have reservations about using the function `stringify` for something like this, as opposed to say `toScratchBlocks`, which I think is much more clear.